website/content/en/status/report-2024-10-2024-12/srcmgr.adoc

```diff
@@ -0,0 +1,14 @@
+=== New srcmgr group
+
+Contact: srcmgr <srcmgr@FreeBSD.org>
+
+A new link:https://www.freebsd.org/administration/#t-srcmgr[source management team] has been appointed by core@ to handle management of the FreeBSD src tree, akin to portmgr@ and doceng@ for the ports and docs trees, respectively.
+The initial members are Ed Maste, Mark Johnston, John Baldwin and Warner Losh.
+srcmgr@ is currently focused on finding ways to make src developers more productive, and to try and manage the large numbers of bug reports and pull requests that we receive.
+The group meets biweekly to discuss src-related issues and spend time triaging bug reports and pull requests.
```

Comment:
Per https://www.merriam-webster.com/dictionary/biweekly, "biweekly" is ambiguous; use either "twice a week" or "every other week" instead of "biweekly".